projects/components/src/assets/resources/en.properties

```diff
@@ -10,6 +10,9 @@ vcd.cc.data-exporter.title=Data Exporter
 vcd.cc.loading=Loading
 vcd.cc.grid.default.pagination={firstItem} - {lastItem} of {totalItems} row(s)
 vcd.cc.unlimited=Unlimited
+vcd.cc.yes=Yes
+vcd.cc.no=No
+vcd.cc.bad.data=The data you have exported contains potential code injection. Would you like to sanitize this data to prevent code injection?
```

Comment:
How about

The data being exported contains information that would cause it to execute formulas if opened with a spreadsheet. Would you like to sanitize this data to prevent code injection?